File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entities/GregorianCalendarEvent.java
package com.byagowi.persiancalendar.entities;

import com.byagowi.persiancalendar.calendar.CivilDate;

public class GregorianCalendarEvent extends AbstractEvent<CivilDate> {
    public GregorianCalendarEvent(CivilDate date, String title, boolean holiday) {
        this.date = date;
        this.title = title;
        this.holiday = holiday;
    }
}

File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entities/GregorianCalendarEvent.kt
package com.byagowi.persiancalendar.entities

import io.github.persiancalendar.calendar.CivilDate

class GregorianCalendarEvent(date: CivilDate, title: String, holiday: Boolean) : AbstractEvent<CivilDate>() {
    init {
        this.date = date
        this.title = title
        this.isHoliday = holiday
    }
}
